### DISCUSSION TITLE
How to publish on Gitlab Pages ?

### DISCUSSION THREAD

**Discussion by Itotutona** [facebook/docusaurus#5160] [Q&A] [answered]
Hello, 
It's not describe how to publish on gitlab pages.Could you help me to explain me how to deploy on gitlab pages?
Thanks

**Comment by slorber** [comment, marked as answer]
Hi

Unfortunately, we can't cover all existing deployment options in an exhaustive way, that's too much work.

You'll have to read gitlab doc to know how to deploy a static site, and adapt that to Docusaurus (ie use `npm run build` and deploy the `build` folder).

This repo seems like a nice example for inspiration: https://gitlab.com/pages/docusaurus/-/tree/master
